### ISSUE TITLE
Accessibility Message Should Be Dismissable

### ISSUE THREAD

**Issue by githubsterer** [keymapperorg/KeyMapper#1484]
**Developer TODO (don't remove)**
- [ ] update documentation

Not sure why the message should be there at all, except momentarily after the setting is made, but in any event should not persist always without any ability to turn it off. The user is alerted when the setting is not set so can presume all is good without having the banner take up precious screen space.

**Comment by jambl3r** [comment]
New proposed UI for 3.0 removes the message and replaces it with a 'Running' status indicator on the top of the screen, see mock-up

![Screenshot_20250224-192102.png](https://github.com/user-attachments/assets/c51cef92-7fe9-40e9-bcd5-9211a023a039)

**Comment by githubsterer** [comment]
Perfecto! Little late to the party so I'm just catching up :-)

    On Monday, February 24, 2025 at 02:21:51 PM EST, Jack Ambler ***@***.***> wrote:  
 
  
New proposed UI for 3.0 removes the message and replaces it with a 'Running' status indicator on the top of the screen, see mock-up

Screenshot_20250224-192102.png (view on web)

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
jambl3r left a comment (keymapperorg/KeyMapper#1484)
New proposed UI for 3.0 removes the message and replaces it with a 'Running' status indicator on the top of the screen, see mock-up

Screenshot_20250224-192102.png (view on web)

**Comment by githubsterer** [comment]
But wouldn't a restore from your friend's backup just replace your existing key maps, or does it just load them into the existing list? If the latter then fine, but if it's the former then that clearly won't help, right?
 
 
  On Mon, Feb 24, 2025 at 6:38 PM, Jack ***@***.***> wrote:   


Closed #1484 as completed.
